src/towncrier/newsfragments/608.feature.rst

```diff
@@ -0,0 +1,2 @@
+Issues are now correctly sorted by issue number even if they have non-digit characters.
```

Comment:
What does "correct" means ?

Maybe something like this

```suggestion
For news fragments that have multiple associted issues, issues are now sorted ascending by issue number even if they have non-digit characters.
```

Reply:
I've clarified it now. This doesn't only affect fragments with multiple associated issues -- it also affects overall sorting of fragments.